vcd/datasource_vcd_nsxt_route_advertisement.go

```diff
@@ -0,0 +1,73 @@
+package vcd
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdNsxtRouteAdvertisement() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdNsxtRouteAdvertisementRead,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"edge_gateway_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "NSX-T Edge Gateway ID in which route advertisement is located",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Define if route advertisement is active",
```

Comment:
```suggestion
				Description: "Defines if route advertisement is active",
```

Reply:
Changed in resource and datasource, thanks!